wcfsetup/install/files/lib/system/view/filter/BooleanFilter.class.php

```diff
@@ -16,6 +16,15 @@
  */
 class BooleanFilter extends AbstractFilter
 {
+    public function __construct(
+        string $id,
+        string $languageItem,
+        string $databaseColumn = '',
+        private readonly bool $reverseValue = false
```

Comment:
This property is unused.

Apart from this extra parameter the signature matches the parent constructor which means that you can remove it.